packages/datagateway-download/src/downloadStatus/downloadStatusTable.component.tsx

```diff
@@ -200,7 +200,14 @@ const DownloadStatusTable: React.FC<DownloadStatusTableProps> = (
         </Grid>
       )}
       <Grid item>
-        <Paper style={{ height: 'calc(100vh - 110px)' }}>
+        {/* Table should take up page but leave room for: SG appbar, SG footer,
+            tabs,table padding, and text above table (respectively). */}
+        <Paper
+          style={{
+            height: 'calc(100vh - 64px -24px - 48px - 48px - (1.75rem + 40px))',
```

Comment:
Not sure if I tested this and then forgot to mention it in my tests review, but the height of the status table appears to have been drastically reduced, see screenshot below.
![image](https://user-images.githubusercontent.com/45173816/104172793-b8124080-53fc-11eb-8f49-301283ca5516.png)

Reply:
I think `calc()` doesn't work with "negative" pixel values, so `- 24px` is fine but `-24px` is breaking it, my fault for not actually testing the change (again).